css/style.css

```diff
@@ -758,6 +1796,124 @@
     color: #fff;
     font-size: 16px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 767px;
+  }
+
+  .wrap-information {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    background-position-y: top;
+    margin-top: 20px;
+  }
+
+  .red-first {
+    max-width: 100%;
+    float: none;
+  }
+
+  .redrow {
+    display: flex;
+    height: 300px;
+    padding-bottom: 3px;
+  }
+
+  .icons {
+    padding: 5px 62px 0 0;
+  }
+
+  .description__screen {
+    padding-bottom: 28px;
+  }
+
+  .img-inch {
+    width: 36px;
+    height: 52px;
+  }
+
+  .data__figure {
+    font-size: 32px;
+    font-weight: 500;
+  }
+
+  .data__display {
+    font-size: 14px;
+  }
+
+  .description__wifi {
+    padding-bottom: 32px;
+  }
+
+  .img-wifi {
+    padding-right: 20px;
+    float: left;
+  }
+
+  .up-date {
+    white-space: nowrap;
+  }
+
+  .description__bag {
+    padding-bottom: 0;
+  }
+
+  .img-cap {
+    width: 36px;
+    height: 50px;
+    padding-top: 6px;
+  }
+
+  .data__space {
+    font-size: 12px;
+  }
+
+  .info-order__img {
+    width: 31px;
+    height: 31px;
+    padding-right: 14px;
+  }
+
+  .info-order__text {
+    font-size: 15px;
+    padding-right: 48px;
+  }
+
+  .wrap {
+    display: flex;
+    justify-content: flex-end;
+    flex-direction: column;
+    background-position: -24px 79px;
+  }
+
+  .details {
+    max-width: 100%;
+    margin-top: 50px;
+  }
+
+  .redrow-scnd {
+    flex-direction: row;
+    display: flex;
+  }
+
+  .title {
+    padding-top: 0;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)